sourced/ml/cmd_entries/repos2bow.py

```diff
@@ -61,6 +62,12 @@ def repos2bow_entry_template(args, select=HeadFiles, cache_hook=None):
     log.info("Calculating the document frequencies...")
     df = uast_extractor.link(BagFeatures2DocFreq()).execute()
     log.info("Writing docfreq to %s", args.docfreq)
+    dirs = os.path.split(args.docfreq)[0]
```

Comment:
You should put this inside Modelforge's `Model.save`; add the new argument `create_missing_dirs` which is `True` by default.